src/main/java/ladder/model/RandomDirectionGenerator.java

```diff
@@ -0,0 +1,13 @@
+package ladder.model;
+
+import java.util.Random;
+
+public class RandomDirectionGenerator implements DirectionGenerator {
+
+    private static final Random random = new Random();
```

Comment:
정적 상수이므로 대문자 컨벤션입니다 😄 
```suggestion
    private static final Random RANDOM = new Random();
```